src/script/cpp_api/s_item.cpp

```diff
@@ -88,6 +88,25 @@ bool ScriptApiItem::item_OnPlace(ItemStack &item,
 	return true;
 }
 
+bool ScriptApiItem::item_OnWield(const ItemStack &item,
+    ServerActiveObject *user)
+{
+	SCRIPTAPI_PRECHECKHEADER
+	
+	int error_handler = PUSH_ERROR_HANDLER(L);
+	
+	// Push callback function on stack
+	if (!getItemCallback(item.name.c_str(), "on_wield"))
+	  return false;
+		
+	// Call function
+	LuaItemStack::create(L, item);
+	objectrefGetOrCreate(L, user);
+	PCALL_RES(lua_pcall(L, 2, 1, error_handler));
```

Comment:
the third argument to lua_pcall here should be `0` since you're not expecting a return value
then the pop below also needs to be changed to `1`

Reply:
Done